modules/nf-core/survivor/merge/main.nf

```diff
@@ -0,0 +1,73 @@
+process SURVIVOR_MERGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::survivor=1.0.7"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/survivor:1.0.7--h9a82719_1':
+        'quay.io/biocontainers/survivor:1.0.7--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(vcfs)
+    val(max_distance_breakpoints)
+    val(min_supporting_callers)
+    val(account_for_type)
+    val(account_for_sv_strands)
+    val(estimate_distanced_by_sv_size)
+    val(min_sv_size)
+
+    output:
+    tuple val(meta), path("*.vcf")   , emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    vcfs.each{
+        if (it.getExtension() == "gz"){
+            error "Gzipped files are not supported by Survivor, please gunzip your VCF files first."
+            // https://github.com/fritzsedlazeck/SURVIVOR/issues/158
+        }
+    }
+
+    """
+    ls *.vcf > names.txt
+
+    SURVIVOR merge \\
+        names.txt \\
```

Comment:
Bit of process substitution to save a write?

```suggestion
    SURVIVOR merge \\
        <(ls *.vcf) \\
```

Reply:
Oh cool I didn't know this was possible, thanks!